packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx

```diff
@@ -0,0 +1,144 @@
+
+import BpkText, { TEXT_STYLES } from '../../bpk-component-text';
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkNavigationTabGroup.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const NAVIGATION_TAB_GROUP_TYPES = {
+  CanvasDefault: 'canvas-default',
+  SurfaceContrast: 'surface-contrast',
+};
+export type NavigationTabGroupTypes =
+  (typeof NAVIGATION_TAB_GROUP_TYPES)[keyof typeof NAVIGATION_TAB_GROUP_TYPES];
+
+type TabItem = {
+  text: string;
+  icon?: FunctionComponent<any> | null;
+  href?: string;
+};
+export type Props = {
+  tabs: TabItem[];
+  type?: NavigationTabGroupTypes;
+  /*
+   * Index parameter to track which is clicked
+   */
+  onItemClick: (tab: TabItem, index: number) => void;
+  selectedIndex: number;
+};
+
+type TabWrapProps = {
+  tab: TabItem;
+  type: NavigationTabGroupTypes;
+  selected: boolean;
+  children: ReactElement;
+  onClick: () => void;
+};
+
+const TabWrap = ({ children, onClick, selected, tab, type }: TabWrapProps) => {
+  const tabStyling = getClassName(
+    'bpk-navigation-tab-wrap',
+    `bpk-navigation-tab-wrap--${type}`,
+    selected && `bpk-navigation-tab-wrap--${type}-selected`,
+  );
+
+  return tab.href ? (
+    <a
+      className={tabStyling}
+      href={tab.href}
+      onClick={onClick}
+      aria-current={selected ? 'page' : undefined}
+    >
+      {children}
+    </a>
+  ) : (
+    <button
+      className={tabStyling}
+      type="button"
+      onClick={onClick}
+      aria-current={selected ? 'page' : undefined}
+      role='link'
+    >
+      {children}
+    </button>
+  );
+};
+
+const BpkNavigationTabGroup = ({
+  onItemClick,
+  selectedIndex,
+  tabs,
+  type = NAVIGATION_TAB_GROUP_TYPES.SurfaceContrast,
+}: Props) => {
+  const [selectedTab, setSelectedTab] = useState(selectedIndex);
+  const handleButtonClick = (tab: TabItem, index: number) => {
+    if (index !== selectedTab) {
+      setSelectedTab(index);
+    }
+    onItemClick(tab, index);
+  };
+
+  const containerStyling = getClassName('bpk-navigation-tab-group');
+
+  return (
+    <nav
+      className={containerStyling}
+      role="navigation"
+      aria-label="Tab Navigation"
+    >
+      {tabs.map((tab, index) => {
+        const selected = index === selectedTab;
+        const Icon = tab.icon;
+        return (
+          <TabWrap
+            key={`index-${index.toString()}`}
+            tab={tab}
+            selected={selected}
+            onClick={() => handleButtonClick(tab, index)}
+            type={type}
+          >
+            <>
+              {Icon ? (
+                <span
+                  className={getClassName(
+                    'bpk-navigation-tab-icon',
+                    `bpk-navigation-tab-icon--${type}`,
+                    selected && `bpk-navigation-tab-icon--${type}-selected`,
+                  )}
+                >
+                  <Icon />
+                </span>
+              ) : null}
```

Comment:
I think you can simplify this without the `if else` by changing to

```suggestion
              {Icon && (
                <span
                  className={getClassName(
                    'bpk-navigation-tab-icon',
                    `bpk-navigation-tab-icon--${type}`,
                    selected && `bpk-navigation-tab-icon--${type}-selected`,
                  )}
                >
                  <Icon />
                </span>
              )}
```

Reply:
Updated in [this commit](https://github.com/Skyscanner/backpack/pull/3599/commits/586eb133545f2614ae5476c20cad35bc6f00f0bd#diff-c6749377497b298af7591bcbc227eebeada4e4022c5b39b373a7ab9b4f77f44eR122)